data-raw/phenotypes.R

```diff
@@ -159,13 +161,198 @@ individual_metadata <- individual_metadata %>%
     mouse_model = as_factor(mouse_model)
   )
 
-## Combine data ----
+### Combine data ----
+
+phenotypes_5xfad <- phenotype_data %>%
+  inner_join(biospecimen_metadata, by = c("individual_id", "specimen_id")) %>%
+  inner_join(individual_metadata, by = "individual_id") %>%
+  select(individual_id, specimen_id, mouse_model, sex, age, tissue, phenotype, units, value)
+
+# 3xTg ----
+
+## Download and read  data ----
+
+# Read file that contains the phenotype name, synapse ID, field to use, and values of Stain to filter for
+phenotype_files <- read_csv(here::here("data-raw", "pathology", "3xTg", "3xTg_data_sources.csv"))
```

Comment:
Another good candidate for a configuration file!

Reply:
Curious what you would put in the config file? It's not as common of a concept in R, I think, so to me, e.g. `3xTg_data_sources.csv` actually is the config file because it lists all the data sources, their versions, filters, etc, rather than listing it out in code (I can see how it might be useful to include the metadata in there too though, as per your other comment!)

Reply:
Here we are hard coding the file format, the name, etc. A configuration file would allow you to read those from an external source.  It would force your code to be general, and read parameters of that file identified by a key.  The advantage to that is that changes to the parameters will not happen in the code, but in this JSON/YAML/etc. file.  It's far easier to update and maintain, and it can save you a lot of time!  It's just a though!  I can show you some examples if you'd like!